example/listProxy/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/haxii/tdns"
+)
+
+var (
+	rpc  = flag.String("rpc", "", "rpc server addr")
+	help = flag.Bool("h", false, "help usage")
+)
+
+func main() {
+	flag.Parse()
+	if *help {
+		flag.Usage()
+		return
+	}
+
+	if *rpc == "" {
+		log.Fatalln("rpc is empty")
+	}
+
+	c := tdns.ConnectClient(*rpc)
+	defer c.Close()
+	proxys, err := c.ListProxyInfo()
```

Comment:
proxies

Reply:
fix all spelling errors, [streetsidesoftware.code-spell-checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) maybe a great plugin for u.